.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+version: 2
+job_defaults: &job_defaults
+    docker:
+        - image: circleci/openjdk:10-jdk-sid-node-browsers-legacy
+    working_directory: ~/scratch
+
+jobs:
+    setup:
+      <<: *job_defaults
+      steps:
+        - checkout
+        - run: mvn dependency:go-offline
+        - save_cache:
+            paths:
+              - ~/.m2
+            key: v1-dependencies-{{ checksum "pom.xml" }}
+
+    validation:
+      <<: *job_defaults
+      steps:
+        - checkout
+        - restore_cache:
+            keys:
+              - v1-dependencies-{{ checksum "pom.xml" }}
+              - v1-dependencies-
+        - run: mvn clean test
+    build:
+      <<: *job_defaults
+      steps:
+        - checkout
+        - restore_cache:
+            keys:
+              - v1-dependencies-{{ checksum "pom.xml" }}
+              - v1-dependencies-
+#        - run: python increment.py VERSION
+        - run: mvn release:clean
+        - run: mvn --batch-mode -Dtag=v$(cat VERSION) release:prepare -DreleaseVersion=v$(cat VERSION)
+    deploy:
+      <<: *job_defaults
+      steps:
+        - checkout
+        - restore_cache:
+        #catching all the dependencies- restore_cache:
+            keys:
+              - v1-dependencies-{{ checksum "pom.xml" }}
+            # fallback to using the latest cache if no exact match is found
+              - v1-dependencies-
+        - run: mvn release:perform
+        - run: apt-get install jq
+        - run: ./post_to_github.sh $API_Token_Github
+
+prod_filter: &prod_filter
```

Comment:
Maybe call it `release_filter` to be more consistent?

Reply:
Will change that, thanks